test/integration/scheduler/framework_test.go

```diff
@@ -456,23 +457,14 @@ func TestPreFilterPlugin(t *testing.T) {
 			},
 		},
 	}
-	// Set empty plugin config for testing
-	emptyPluginConfig := []schedulerconfig.PluginConfig{}
 
 	// Create the master and the scheduler with the test plugin set.
-	context := initTestSchedulerWithOptions(t,
-		initTestMaster(t, "prefilter-plugin", nil),
-		false, nil, registry, plugins, emptyPluginConfig, time.Second)
-
+	context, cs := initTestSchedulerForFrameworkTest(t, initTestMaster(t, "prefilter-plugin", nil), 2,
+		scheduler.WithFrameworkPlugins(plugins),
+		scheduler.WithFrameworkRegistry(registry),
+		scheduler.WithFrameworkConfigProducerRegistry(nil))
```

Comment:
I assume we want a clean framework (none from priorities or predicates) for all these tests?
We can actually add the option in `initTestSchedulerForFrameworkTest`

Reply:
WDYT?

Reply:
sorry I missed it, done.